01/app.js

```diff
@@ -1 +1,11 @@
-console.log('DOM');
\ No newline at end of file
+console.log('DOM');
+
+const liEl = document.querySelector('.comments__item.comments__item--newest');
+
+if (liEl) {
+    const liElAttribute = liEl.querySelectorAll('[data-info]');
+
+    if (liElAttribute) {
```

Comment:
Tutaj `if` nic nie da i jest nie potrzebny. `.querySelectorAll` zwraca tablicę, która jest obiektem.
Obiekt zawsze jest "prawidziwy" czyli `if( [] )` lub `if( {} ) ` - zawsze daje prawdę, nawet bez zawartości.

Dlatego jak już to: `if (liElAttribute.length > 0)`

Reply:
Dzięki za uwagę – zorientowałem się dopiero w późniejszych taskach, że przy liście elementów nie ma sensu robić sprawdzenia :D.